packages/storefront/src/views/Page.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <p-tabs-bar v-if="hasTabs" :active-tab-index="activeTabIndex" size="medium">
+    <p-tabs-bar :theme="$store.getters.platformTheme" v-if="hasTabs" :active-tab-index="activeTabIndex" size="medium">
```

Comment:
Usually structural directives are listed first for better readability and consistency. 
```suggestion
    <p-tabs-bar v-if="hasTabs" :theme="$store.getters.platformTheme" :active-tab-index="activeTabIndex" size="medium">
```